eventually/team/models.py

```diff
@@ -0,0 +1,124 @@
+"""
+Team model
+=======================
+
+This module implements class that represents the team entity.
+"""
+from django.db import models
+from django.db import IntegrityError
+
+# Create your models here.
+
+
+class Team(models.Model):
+
+    """
+    .. class:: Team
+    Describing of team untity.
+    Attributes:
+    ===========
+
+        :param name: Name of the certain team.
+        :type name: string
+        :param description: Describing core information of the team.
+        :type description: string
+        :param image: Describing link to image of the team.
+        :type description: string
+        :param create_date: The date when the
+                            certain team was created.
+        :type: create_date: datetime
+        :param update_date: The date when
+                    the certain team was last time edited.
+    """
+    name = models.CharField(max_length=30)
+    description = models.TextField(null=True)
+    image = models.CharField(max_length=300, null=True)
+    # owner_id= models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
delete commented code.
in the repository, it should not be.

Reply:
Done